### PR TITLE
removing whitespaces and adding comment

### PR DESCRIPTION
**Pull request by nicotela**

### CODE REVIEW

File: uloha_2.py [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
+def line_counter(path_to_file):
     with open(path_to_file) as f:
         text = f.read()
     count_lines = text.count("\n") + 1
     return count_lines
 
 result = line_counter("/Users/nicolgotzova/Documents/Ukoly/Data/data2.txt")
-print (result)
+print(result)
+#comment
```
Comment (tomas-danek) [inline uloha_2.py]: please remove this useless line

File: uloha_2.py [hunk 2]
```diff
@@ -1,8 +1,9 @@
-def line_counter (path_to_file):
+def line_counter(path_to_file):
```
Comment (tomas-danek) [inline uloha_2.py:1]: nice beautification of code :)

**Review by tomas-danek — CHANGES_REQUESTED** [review]
Overall nice work, please see comments

File: uloha_2.py (hunk 1 above)
Comment (nicotela) [reply uloha_2.py]: removed

**Review by tomas-danek — APPROVED** [review]
nice work :)

**Pull request merged**